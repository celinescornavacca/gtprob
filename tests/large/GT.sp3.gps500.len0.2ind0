((((((((((B111,B112),((B170,(B399,B400)),B85)),((B44,((B421,B422),B353)),(B23,((B200,(B251,((B361,B362),B332))),(B86,(B106,((B236,((B300,B301),B256)),(B438,B439)))))))),(B71,(B423,B424))),((((B156,(B371,B372)),(B363,B364)),B0),(((B342,B343),B320),B220))),(((((((B245,B246),(B430,B431)),((B133,B134),B8)),(((B355,B356),(B147,(B207,B208))),B16)),(((((B202,B203),B201),B160),(B393,B394)),B24)),((((((B335,B336),((B153,B154),B114)),(B377,B378)),((B314,B315),((B69,B70),(B45,B46)))),(((B440,(B492,B493)),(((((B357,B358),B313),B52),(((B97,((B159,((B443,B444),B327)),((B373,B374),B181))),(B266,B267)),(B276,B277))),((((B55,((B472,B473),B96)),B53),((B391,B392),(B403,B404))),B37))),B3)),((((B177,B178),(B94,B95)),((((((B449,B450),(B359,B360)),(B461,B462)),(B171,((B397,B398),((B307,(B334,(B344,B345))),B227)))),(B405,B406)),B49)),((((B67,(((B308,B309),(B179,(B432,B433))),(B75,B76))),((B197,B198),(B29,(B98,B99)))),B13),((B47,((B90,(B447,B448)),B57)),(B72,((((B282,B283),B230),B172),((B318,B319),(B288,B289))))))))),(((B183,B184),B9),((B1,(B136,B137)),(((B413,B414),((((B366,B367),B231),B128),B118)),(B30,(B31,(B33,B34)))))))),(((((((B253,(B417,B418)),((((B292,B293),B101),((B150,B151),(((((B468,B469),B265),B58),(B91,(B247,B248))),B21))),((B254,B255),(B304,B305)))),((((((B135,(B290,B291)),(B195,B196)),B20),B15),((B152,(B347,(B379,B380))),B35)),((B164,B165),(((B425,B426),B257),B32)))),(((((B330,B331),(B476,B477)),(B186,B187)),((((((B274,B275),B130),(B223,B224)),((((B92,B93),(B312,(B321,B322))),(B132,(B306,(B455,B456)))),((B63,(B323,B324)),B41))),(((B352,(B466,B467)),(((B121,B122),B48),(B68,(B129,(B369,B370))))),(B296,B297))),(((B484,B485),(B340,B341)),B115))),((((((B213,B214),(B310,B311)),((B139,B140),((((B480,B481),(B395,B396)),((B387,B388),(B209,B210))),B56))),(B110,(B333,(((B411,B412),(B453,B454)),(B464,B465))))),(B211,B212)),((B100,(B204,B205)),(((B138,(B385,B386)),(B119,B120)),((B108,B109),B36)))))),((B6,(B104,B105)),(B182,(B474,B475)))),((((B166,B167),(B206,(B490,B491))),(B7,((B189,B190),((B54,(B141,B142)),(B199,((B337,B338),(B260,B261))))))),((((((((B389,B390),B237),(B240,B241)),(((((B272,B273),(B434,B435)),B62),(B116,B117)),(B258,B259))),B10),(((B263,B264),((B383,B384),(B216,B217))),B12)),(((B82,(B193,B194)),B73),B42)),((B4,((B325,B326),B215)),(((B185,(B494,B495)),B14),(B219,(B232,B233))))))),(((((B457,B458),B174),((B27,((B64,(B78,B79)),(B80,B81))),(B50,(B59,(B478,B479))))),((((B87,B88),B74),((B83,B84),(B168,B169))),(B428,B429))),((((B145,B146),(B157,B158)),(B218,(B436,B437))),((((B482,B483),(((B375,B376),B226),(B294,B295))),((B28,(B441,B442)),((B65,B66),B38))),(((B5,((B350,B351),B22)),(((B328,B329),(B102,B103)),(B131,((B234,B235),B163)))),(B11,((((B415,B416),(((B238,B239),B155),((B180,(B451,B452)),B127))),(B113,(B148,B149))),(B316,B317))))))))),(B2,(((((((B77,(B278,B279)),((((B427,(B445,B446)),(B191,B192)),(((B470,B471),B463),(B280,B281))),(((B419,B420),((B268,B269),(B270,B271))),(B242,B243)))),(B126,(B188,((B486,B487),B368)))),(((((((B409,B410),B365),B299),(B175,B176)),(B221,B222)),((B107,((B302,B303),B262)),(B228,((B252,((B346,(B496,B497)),B339)),B244)))),B19)),((((B89,((B284,B285),(B348,B349))),((B298,(B488,B489)),B173)),((B124,B125),(B43,((B401,B402),B354)))),(B17,B18))),(((B161,B162),(B459,B460)),((B25,B26),(B39,B40)))),((((((B407,B408),(B381,B382)),B229),(B498,B499)),B51),(((B60,B61),((B143,B144),B123)),(((B286,B287),B225),(B249,B250))))))),(((((((C370,C371),C25),((C84,(C119,((C341,C342),C321))),(((C472,C473),((C254,(C364,(C494,C495))),(C136,C137))),(((((C271,((C428,C429),C347)),(C354,(C414,C415))),C35),((C289,(C317,C318)),C217)),(((C250,C251),((C407,C408),C131)),C48))))),((((((C350,C351),(C166,C167)),C42),(C403,C404)),C4),((((C284,C285),(C240,C241)),((C246,(C376,C377)),C198)),(C17,(C432,C433))))),(((((((C143,(C394,(C468,C469))),((C335,C336),(C450,C451))),(C104,C105)),((C40,C41),C30)),(((((C228,C229),(C189,C190)),(C16,((((((C476,C477),C183),C79),(C57,C58)),C26),((C53,((C299,C300),(C239,(C278,((C440,C441),C437))))),(((C192,(C417,C418)),C157),(((C492,C493),C235),C96)))))),((C31,C32),(C77,C78))),((((C110,(((C315,C316),C249),(C274,C275))),C100),(C296,C297)),((C290,C291),C126)))),(C24,(C43,C44))),((((C134,C135),(((((C446,C447),C416),C219),((C322,C323),C118)),C11)),(((C345,C346),(((C328,C329),C71),C45)),(C454,C455))),(C5,(C36,C37))))),(((((C230,((C448,C449),C359)),((((C233,C234),((C306,C307),C162)),((C294,C295),C132)),(C175,C176))),(((((C144,C145),C138),C90),C68),(((C438,C439),C369),(C203,C204)))),(((C124,C125),(((((C66,(C367,C368)),C29),C28),(C456,C457)),((C101,C102),(C314,(C333,C334))))),(((C59,((C181,C182),(C164,C165))),C34),((((C72,C73),C67),(C186,((C480,C481),C419))),((C223,C224),((C474,C475),C201)))))),((((C362,C363),((((C435,C436),C94),((C259,C260),(C484,C485))),C3)),((((C180,(C355,(C380,C381))),C113),((((C106,C107),C103),(C490,C491)),(((C193,C194),C141),(C99,(C120,((C282,C283),C150)))))),(((C218,(C411,C412)),(C172,(C232,((C383,C384),C255)))),(((C70,((C401,C402),(C326,C327))),(((C460,C461),C214),C65)),((((((C247,C248),((C301,(C308,(C311,(C343,C344)))),C151)),C80),C49),(C261,C262)),((C108,C109),C18)))))),((C1,((((((C276,C277),C129),(C114,(C257,C258))),(((C256,(C388,C389)),C122),(C160,C161))),(((((((C69,((((C187,C188),C139),((C191,(C220,C221)),C149)),((C168,C169),C93))),C33),C15),C14),C10),C9),(C21,(C348,C349)))),(C55,((C365,C366),C202)))),C0)))),((((((C20,(((((C356,C357),(C163,(C337,C338))),C91),(C170,C171)),((((C426,C427),(C391,(C470,C471))),C85),C62))),((C87,C88),((((C130,((C352,C353),C133)),(C372,C373)),C56),((C184,C185),C154)))),((C225,(C263,C264)),((C244,C245),C209))),((C498,C499),C23)),((((C397,C398),(C252,C253)),(((C399,C400),(((C140,(C482,C483)),((C74,((C395,C396),C81)),((C173,C174),C156))),(C51,(C177,C178)))),((C210,(C238,((C387,((C486,C487),C442)),(C287,C288)))),(((C409,C410),((C392,C393),(C212,C213))),((((((C242,((C422,C423),C243)),(C466,C467)),(C116,C117)),C75),((C413,((C462,C463),C434)),C121)),((((((C458,C459),C406),C405),C123),((C279,(C286,((C424,C425),C390))),C179)),(C97,(C197,(C199,C200))))))))),((((C52,(C272,C273)),C7),(((C292,C293),C76),(C12,((C54,(C64,((C215,C216),C115))),((C339,C340),C98))))),(((C142,(C146,C147)),((((C420,C421),C305),C50),(C95,(C153,((C444,C445),C324))))),((C378,C379),(C6,((C206,C207),(C60,C61)))))))),((((C205,(C226,C227)),(((((C92,(C385,C386)),(C111,C112)),(C478,C479)),(((C303,C304),(C269,C270)),(((C236,C237),(C222,(C464,C465))),C22))),(C8,(((C27,(C82,C83)),(C312,C313)),((((C488,C489),C443),C358),C325))))),((C13,(((((C319,C320),C155),C46),(C86,(C298,(C430,C431)))),((C208,(C267,C268)),((C152,(((C265,C266),(C302,(C309,C310))),C231)),(C374,C375))))),C2)),(((C63,((C148,((C382,(C452,C453)),C330)),C89)),C47),(((((((C331,C332),(C280,C281)),((C158,C159),(C38,C39))),(C360,C361)),(C127,C128)),((C496,C497),C211)),((C195,C196),C19))))))),(((((A395,A396),(((A248,(A266,A267)),A118),(A200,A201))),(((((((A45,A46),(((A276,A277),A234),A183)),(A34,(((A101,(A127,(A151,(A169,A170)))),(((A194,(A260,A261)),A144),A91)),(A149,(A393,A394))))),((A103,(A211,(A348,A349))),((A247,((A413,A414),(A357,A358))),A134))),((((A5,((A407,A408),(((A456,A457),A72),(A445,A446)))),(A99,A100)),(((A263,A264),((A206,A207),(A235,((A336,(A449,A450)),A254)))),(A2,((((A136,((A464,A465),(A391,A392))),(A494,A495)),(A342,(A405,A406))),((A240,A241),A77))))),((A3,(((A131,A132),(A308,A309)),(A21,A22))),(A4,(A14,((A271,A272),A80)))))),((((A86,A87),((((A462,(A466,A467)),A397),A354),A121)),(A129,(A381,A382))),(((A296,(A329,(A373,A374))),(A115,A116)),((A482,A483),A12)))),(((A38,A39),(A198,A199)),(((((((A377,A378),A60),(A23,(A167,A168))),(((A468,A469),A320),A94)),((((A142,A143),(A40,A41)),((A249,A250),(A238,A239))),((A297,A298),(((A68,A69),(A191,A192)),(((A165,A166),((A463,(A484,A485)),A156)),A119))))),(((A49,((A470,A471),A257)),A35),((((A159,A160),(A205,(A486,A487))),A28),(A84,A85)))),(A8,(((A417,A418),(A189,A190)),((A230,A231),(A163,(A251,(A424,A425)))))))))),((A7,((A474,A475),((A292,(A411,A412)),A252))),(A50,(A171,A172)))),((((((((A311,A312),A193),(A213,(A288,A289))),(((A242,A243),(((((A447,A448),A423),A372),((A387,A388),A214)),A137)),(A58,(A472,A473)))),(A216,A217)),((((A92,A93),A18),((A182,(A278,A279)),((A399,(A415,A416)),(((A419,A420),A128),A96)))),(A228,A229))),((((A343,A344),A221),(A370,A371)),(A31,A32))),(((((A44,(A108,(((A359,A360),A265),A133))),A26),((A125,A126),((A323,A324),((A375,A376),(A361,A362))))),(((A71,(A327,A328)),(((((A280,(A460,A461)),(A222,A223)),A120),A43),(A56,A57))),((A439,A440),((A401,A402),((A107,(A498,A499)),(A454,A455)))))),(((((((((A10,(((A443,A444),(A95,((A437,A438),A97))),(((A355,A356),A114),(A337,A338)))),((A112,((A269,A270),A157)),(A496,A497))),(((((A78,A79),A73),(A208,A209)),(A19,A20)),(((((A232,A233),A55),(A52,(A81,(A180,A181)))),A33),((A130,((A316,A317),(A244,A245))),((A53,(((A135,(A313,A314)),A90),A64)),A47))))),((A187,A188),((A331,A332),A15))),((A177,A178),(A24,(A82,A83)))),((((A430,A431),A51),(A173,(A294,A295))),A48)),(((((((A476,A477),A345),A306),(A30,(((A435,A436),A333),A74))),((((A113,(((A218,(A281,A282)),(A185,A186)),(((A283,A284),A273),(A224,A225)))),(A37,((((A154,A155),((A383,A384),A152)),(A75,A76)),A42))),(A59,(A110,A111))),(((A285,A286),A109),(A274,A275)))),((A303,A304),(A102,((A400,(A432,A433)),A138)))),(A1,((((A65,((A203,A204),A117)),(A29,((A409,A410),(A441,A442)))),(((((A488,A489),A434),A307),((A88,A89),((A141,(A368,A369)),(A150,(A318,A319))))),A11)),(((((A290,A291),A70),(A330,(A364,A365))),((A158,(A346,A347)),(A36,(A301,A302)))),((A403,A404),((A226,((A458,A459),A310)),(((A428,A429),A287),(((((A255,A256),A179),(A490,A491)),A61),(A122,A123)))))))))),(((((A219,A220),A210),A9),((((((A253,(A339,A340)),A197),(A227,(A246,(A352,A353)))),(A202,((A389,A390),A268))),(A6,((A350,A351),(A299,A300)))),(((A124,(A174,(A341,(A398,(A421,A422))))),((A236,A237),((A139,A140),(A321,A322)))),((A161,A162),((A27,((A147,A148),((A175,A176),(A184,(A366,A367))))),A17))))),A0)),((((A164,(A195,A196)),(A334,A335)),(((A66,A67),A54),A13)),((((((A98,(A104,((A478,A479),A363))),A25),(A62,A63)),(((((A452,A453),A451),A315),A153),(A262,(A385,A386)))),(A16,(A105,A106))),(((A145,A146),(((A293,(A426,A427)),((A258,A259),A215)),(((A492,A493),(A379,A380)),(A305,(A480,A481))))),((A325,A326),A212)))))))));
