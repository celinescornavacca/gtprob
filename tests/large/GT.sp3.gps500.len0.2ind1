((((A0,(((((A445,A446),(((A431,A432),A412),A310)),((((((A274,A275),(A366,A367)),A151),(A246,(A294,A295))),A57),((A471,A472),(A441,A442)))),((((A109,A110),(A106,(A393,A394))),((A29,(A190,A191)),((A49,A50),A25))),((((A161,(A188,A189)),((A469,A470),A26)),((A21,((A473,A474),A233)),(A28,(((A235,A236),A176),(A69,A70))))),(A138,A139)))),(A2,((((A87,A88),((A451,A452),((A318,(A354,A355)),(A45,A46)))),((A154,(((A183,((A395,A396),A237)),A181),A180)),(A304,A305))),A3)))),(((((((A166,A167),(A134,(A268,A269))),A91),A15),A4),(((A377,A378),(A248,(A429,A430))),(((((A194,A195),(A155,((A298,A299),A199))),(A67,A68)),(A301,A302)),A7))),((((((A92,((A481,A482),A182)),((A147,(A489,A490)),(A358,A359))),((A231,A232),((A107,((A447,A448),A283)),(A306,A307)))),(((((A103,(A439,A440)),(A281,A282)),((A459,A460),(A141,A142))),(A242,A243)),((((((A27,(A330,A331)),A17),((A196,A197),(A423,A424))),(((((A143,(A383,A384)),A76),(((A360,A361),A252),A187)),(((A78,A79),A31),A24)),((A38,((A372,A373),((A496,A497),(((A453,A454),A163),(A169,(A177,((A272,A273),(A319,A320)))))))),((((A316,A317),A221),(A391,A392)),(A58,A59))))),((((A362,A363),(A234,(A352,A353))),A89),A44)),((A127,A128),(A223,A224))))),(((A449,A450),((A419,A420),A327)),(((((A347,(A475,A476)),(((((A152,A153),(((A467,A468),A321),A175)),(A216,(A222,(A261,A262)))),((A51,((((A465,A466),A336),(A129,(A437,A438))),(A254,(A350,A351)))),(A18,A19))),((((A435,A436),((A123,A124),A94)),(((A313,(A332,A333)),A115),A43)),(A118,A119)))),A6),(((A227,A228),A132),(((A214,A215),((A255,A256),A184)),A9))),((A97,A98),(((((((A84,((A495,(A498,A499)),(((A251,((A385,A386),A267)),(A178,A179)),(A165,(A397,A398))))),(A61,A62)),((A399,A400),A81)),A23),(((A125,(A206,A207)),((A172,(A461,A462)),(A130,A131))),A60)),(A102,(A342,A343))),(((A244,A245),A156),(A212,A213))))))),(((((A344,(A410,A411)),(A219,A220)),A35),A34),(((A173,A174),A14),((A53,((A311,A312),((A144,A145),A95))),(A292,A293))))))),(((((((A217,A218),((A229,A230),(A210,A211))),((((((((A198,((A374,(A375,(A491,A492))),A263)),A170),A126),(((A433,A434),A337),(A368,A369))),((A208,A209),(A52,(A90,((A323,A324),A137))))),(A36,(((A266,(A279,A280)),((A300,(A479,A480)),A247)),A73))),A5),(((A150,(A285,A286)),(((A389,A390),A168),A164)),((((A202,A203),((A328,A329),(A381,A382))),(A47,A48)),((((A314,A315),A133),((A425,A426),A413)),((((((A277,A278),(A148,A149)),(A157,A158)),A39),(((A75,((A406,(A408,A409)),A93)),(A443,A444)),A13)),A8)))))),((((((A72,(((A379,A380),A376),((A289,(A421,A422)),(A345,A346)))),(((A65,((A260,(A287,A288)),A71)),A30),(A82,A83))),((A20,((((A257,(A340,A341)),(A238,A239)),(((A200,A201),((A405,(A487,A488)),(A364,A365))),(A325,A326))),(A22,(A64,(A116,A117))))),A10)),(A11,A12)),(((((A171,(A457,A458)),A41),((A284,(A455,A456)),((((A348,A349),(A338,A339)),A140),A63))),((A270,A271),(A485,A486))),(((A322,((A407,(A477,A478)),(A417,A418))),(A258,A259)),(((A80,(A108,(A264,A265))),(A104,A105)),((((A192,A193),A162),(A113,A114)),(A42,(A54,A55))))))),(((((A99,(A205,(A356,A357))),(A111,A112)),(A100,A101)),((A204,(A401,A402)),A77)),(((A56,(A74,(A122,(A290,A291)))),((((A483,A484),(A416,(A463,A464))),(A185,A186)),(A427,A428))),(A253,(A308,A309)))))),(((A32,A33),((A249,A250),(A276,(A493,A494)))),(((A96,(A414,A415)),A37),((A334,A335),(A303,(A387,A388)))))),(((A159,A160),(((A120,A121),((A370,A371),(A66,(A135,A136)))),(A85,A86))),(((A225,A226),A40),((A240,A241),((A403,A404),A146))))),(A1,((A296,A297),A16)))),(((((((((B251,(B287,B288)),(B45,((((B242,((B436,B437),B427)),(B269,(B367,B368))),(B175,(B223,B224))),(((B488,B489),B197),B132)))),B23),(B26,(((B310,(B411,B412)),B279),((B336,B337),B216)))),((((((B478,B479),(B470,B471)),B93),B84),B62),(((B222,(B384,B385)),(B200,B201)),((((B285,B286),B135),B127),B73)))),((((B211,B212),(B162,((B440,B441),B350))),(B102,((B376,B377),(B190,B191)))),((((B81,((B355,B356),B120)),((B371,B372),(B284,(B369,B370)))),(((B324,B325),((B484,B485),B184)),(B114,B115))),(((B91,(B292,B293)),(((B480,B481),B123),((B258,B259),(B311,(B494,B495))))),(B378,B379))))),(((B386,B387),(B202,B203)),(((((((((((B50,(B59,(B315,(B410,(B450,B451))))),(B60,B61)),((B168,B169),B149)),(B2,B3)),(((((B490,B491),B245),B34),(B53,((B395,B396),((B332,B333),B161)))),(((B134,(B419,B420)),((B391,(B403,B404)),((B454,B455),B164))),B16))),B0),((B105,(B205,B206)),(B12,(B124,B125)))),(((B256,B257),B97),(((B41,B42),(((B110,B111),B98),((B83,(B198,B199)),B67))),((B43,B44),(((B358,B359),((((B389,B390),B221),((B322,B323),B217)),B133)),(B20,B21)))))),((((((((B272,B273),B167),(B466,B467)),(((B142,(B401,B402)),B65),((B74,B75),((B296,B297),(((B342,B343),((B281,B282),B260)),B196))))),(B280,(B446,B447))),((((B375,(B476,B477)),(B215,(B334,B335))),((B7,((B327,B328),B14)),((B80,(B330,B331)),((((B452,B453),B204),(B263,B264)),(B298,B299))))),(B176,(B432,B433)))),B1),((B85,B86),(B270,((B434,B435),(B415,B416)))))),(((B460,B461),B312),((((B458,B459),B72),(((B143,B144),((B308,B309),(B306,B307))),(B319,(B486,B487)))),B19))),(((((B130,B131),(((B388,(B430,B431)),B329),((B64,(((B188,B189),(B267,B268)),(((B275,B276),(B301,B302)),B141))),(B106,B107)))),(B192,B193)),((((((B344,B345),B246),B87),((((B261,B262),B89),((B117,B118),(B138,(B338,B339)))),(((B150,B151),B88),(((B157,(B170,B171)),(B305,((B405,B406),B314))),((B173,B174),B63))))),((B28,(B57,B58)),(B51,B52))),(((B36,B37),(B207,B208)),((B95,B96),((((B492,B493),B249),B209),B104))))),(((((((((B320,B321),B166),((B112,B113),B94)),(B237,B238)),(B15,((B496,B497),((B153,B154),((B373,B374),B128))))),((B468,B469),(B11,(((B229,(B340,B341)),(B48,B49)),((B346,B347),B30))))),(((((B382,B383),B82),((B121,(B172,(B185,B186))),(B70,(B228,(B423,B424))))),((B283,(B413,B414)),((B303,B304),((B234,B235),B183)))),(((B5,(B9,(B140,(B194,B195)))),(B6,((B17,B18),((B71,(B213,B214)),((B247,B248),((B353,B354),B236)))))),(B4,(((B472,B473),B92),((B277,B278),B274)))))),(B10,(((B474,B475),(B32,B33)),(((B254,B255),B210),(B421,B422))))),((B482,B483),(((((B456,B457),B101),B38),B27),((B76,B77),(B68,B69))))))))),((((((((B448,B449),B362),(((B393,B394),B129),(B241,(B294,B295)))),((B181,B182),B25)),(B351,B352)),((((B136,B137),B29),(((B116,(B155,B156)),(((B289,(B408,B409)),B220),B187)),B8)),((B417,B418),((B39,B40),((((B99,((B271,(B397,B398)),B250)),(B78,B79)),((B444,B445),B300)),(((B177,B178),B119),((B139,((B357,((B464,B465),B407)),(((B244,(B442,B443)),(B290,B291)),(B313,((B365,B366),B326))))),B90))))))),((((((B218,B219),(B126,(B233,(B363,B364)))),((((B54,(B179,B180)),B22),((B147,B148),(((B425,B426),(B231,B232)),B163))),((B35,(((B226,B227),(((B318,(B428,B429)),B225),B159)),(B265,B266))),(B158,(B360,B361))))),(B13,(((B55,B56),(B66,((B380,B381),((B348,B349),B243)))),((B103,(B160,(B239,B240))),B31)))),(((B165,(B438,B439)),(B498,B499)),(B46,B47))),((B108,B109),((B462,B463),((B399,B400),B392))))),((((B145,B146),((B122,(((B252,B253),B230),B152)),B100)),(B316,B317)),B24))),(((((((C353,C354),C82),C65),((C179,C180),((((((((C228,C229),C136),C89),(C68,C69)),(C114,(C217,C218))),C39),(C430,C431)),(((((C426,C427),C207),C201),((C251,((C404,C405),C398)),C225)),(C157,(C480,C481)))))),(((((C441,C442),C186),(C116,((C197,(C262,(((C437,C438),C384),(C281,C282)))),(C256,(C424,C425))))),((((C119,C120),((C266,(C311,(C453,C454))),C92)),(((((C462,C463),C457),C263),(((C103,C104),(C159,(C304,C305))),C93)),((C267,C268),C70))),C8)),(((C490,C491),C255),(C236,C237)))),((((((C153,C154),(C402,C403)),((C171,C172),(C269,C270))),((C5,(((C137,C138),C55),((C15,(((C320,C321),C245),C81)),((C259,C260),C185)))),((C112,C113),(C199,C200)))),((((C127,C128),C125),(((C287,C288),C181),(C252,(C466,C467)))),((C130,(C164,(C253,C254))),(((C110,C111),((C85,C86),((C474,C475),(C476,C477)))),(C76,C77))))),((C9,(C44,(C52,(((C176,C177),(C364,C365)),(C188,C189))))),((((((C109,(C226,C227)),C12),(C96,((C219,C220),(C208,C209)))),C4),((C264,C265),((C22,C23),C21))),C1)))),(((((C2,(C6,(C27,((C313,(C435,C436)),C212)))),((((C324,C325),((C486,C487),C117)),C105),(C482,C483))),((C129,(C173,((C478,C479),(C283,C284)))),((C24,((((C272,C273),C261),(C276,(C350,(C360,C361)))),((C48,(C83,C84)),((C183,C184),C139)))),(C30,C31)))),(((C32,((C472,C473),(C243,C244))),((C376,(C410,C411)),C234)),((((((C492,C493),(C445,C446)),C121),C45),(C17,(((C277,C278),(C250,(C420,C421))),(C78,(C107,C108))))),(((C42,(C178,(C213,C214))),((C385,C386),(C460,C461))),(((C348,C349),((C297,C298),(C332,C333))),((C346,C347),C274)))))),(((((((C291,C292),C148),(C443,C444)),(((C101,C102),C99),(C494,C495))),(((((C322,(C372,C373)),C64),C37),((C58,(C66,C67)),(C142,((C458,C459),(C182,(C362,C363)))))),(C28,C29))),((((((C205,C206),C191),(C357,C358)),C168),(((((C230,C231),(C34,((C80,(((C412,C413),C406),((C488,C489),C271))),((C336,C337),C249)))),(C7,((((C222,C223),(C160,C161)),(((C257,C258),C204),C134)),(C144,(C235,(C382,C383)))))),(C33,(C49,((C464,C465),(C455,C456))))),((((((C447,C448),C345),C20),C13),((C10,((C75,((C396,C397),(C285,C286))),C43)),(C87,C88))),((C195,((C416,C417),(C334,C335))),(((C323,(C407,C408)),(C135,(C240,C241))),(C94,C95)))))),(((((C302,C303),(((C470,C471),((C318,C319),C299)),C224)),(C132,(C341,C342))),((((C355,C356),(C374,C375)),(C165,C166)),(C122,(((C309,C310),C158),C143)))),((((C289,C290),(C155,C156)),(((C38,(C47,((C59,C60),((C377,(C390,C391)),(C79,(((C388,C389),C387),((C468,C469),C233))))))),((((C314,C315),C106),(C97,(C306,C307))),(((C484,C485),(C366,C367)),((C74,(((C242,(C339,C340)),C126),(C133,((C370,C371),C211)))),(C90,C91))))),(((C196,(C247,C248)),C124),((C53,C54),C36)))),C0)))),((((C368,C369),(C131,(C238,C239))),((((((C449,C450),(C428,C429)),((C246,(C401,(C414,C415))),C118)),((C18,C19),C16)),(((C50,C51),(C162,C163)),C11)),(((C56,C57),((C170,(C174,C175)),(((((C338,(C432,C433)),C312),(C394,C395)),C169),(C380,C381)))),((((C279,C280),(C202,C203)),C192),(C359,(C392,C393)))))),((((C46,((C351,C352),((((C193,C194),C167),C141),C71))),(((C14,((C328,C329),C221)),((C275,(C399,C400)),C190)),(C3,(C72,C73)))),((((((((C300,C301),C232),C210),C115),(((C378,C379),C294),(C61,C62))),(((((C434,(C451,C452)),C409),(C145,C146)),((((((C295,C296),(C343,C344)),C140),(C439,C440)),((C149,C150),((C151,C152),C123))),C35)),(((C147,(C308,(C498,C499))),(C215,C216)),C25))),(C63,(C496,C497))),(C330,C331))),(((C100,((C198,(C422,C423)),(C326,C327))),(C40,C41)),((C98,((C187,(C293,(C418,C419))),(C316,C317))),C26)))))))));
