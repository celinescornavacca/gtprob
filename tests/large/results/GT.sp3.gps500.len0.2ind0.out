-8613.01105855
